extensions/postgres-flyway/src/main/resources/db/migration/V10__add_contract_termination.sql

```diff
@@ -0,0 +1,25 @@
+--
+--  Copyright (c) 2024 sovity GmbH
+--
+--  This program and the accompanying materials are made available under the
+--  terms of the Apache License, Version 2.0 which is available at
+--  https://www.apache.org/licenses/LICENSE-2.0
+--
+--  SPDX-License-Identifier: Apache-2.0
+--
+--  Contributors:
+--       sovity GmbH - initial API and implementation
+--
+
+CREATE TYPE contract_terminated_by AS ENUM ('SELF', 'COUNTERPARTY');
+
+create table sovity_contract_termination
+(
+    contract_agreement_id varchar primary key,
+    reason                varchar                  not null,
+    detail                varchar                  not null,
```

Comment:
![image](https://github.com/user-attachments/assets/1a539b9e-ae38-4a6a-a95c-1ef6163fead5)

Reply:
Yeah, it's the convention to use text with postgresql until you very clearly need something else.

Since if you want for example text[] to map to List<String> you have to be very specific with the type name, we should use text.

Reply:
Switched to `text`.